docs/OncoTree-Version-Converter.md

```diff
@@ -0,0 +1,80 @@
+# OncoTree Version Converter Tool
+
+The OncoTree Version Converter tool was developed to facilitate forwards- and backwards-mapping of OncoTree Codes between OncoTree release versions. Instructions for running the tool can be found below in the [Running the tool](#running-the-tool) section. Background information on the Oncotree history modeling and examples of use cases can be found in the [OncoTree History Modeling](#oncotree-history-modeling) section.
+
+## [Running the tool](#running-the-tool)
+
+The OncoTree Version Converter tool can be run with the following command:
+
+
+```
+python <path/to/scripts/convert_oncotree_version.py> --input-file <path/to/input/file> --output-file <path/to/output/file> --source-version <source_oncotree_version> --target-version <target_oncotree_version> --oncotree-url <oncotree_instance_url>
+```
+
+**Options**
+- `-i | --input-file`: This is the input clinical file path. It must contain `ONCOTREE_CODE` in the file header and it must contain OncoTree codes corresponding to the `<source_oncotree_version>`. Read more about the cBioPortal clinical file format [here](https://docs.cbioportal.org/5.1-data-loading/data-loading/file-formats#clinical-data).
+- `-o | --output-file`: This is the path to the output clinical file that will be generated. It will contain mapped OncoTree codes from `<source_oncotree_version>`-to-`<target_oncotree_version>` and a new column called `ONCOTREE_SUGGESTIONS` containing suggestions for OncoTree codes if a direct mapping was not available.
+- `-s | --source-version`: This is the source OncoTree version. The OncoTree codes in the input file must correspond to this version.
+- `-t | --target-version`: This is the target OncoTree version that the script will attempt to map the input file OncoTree codes to.
+- `-u | --oncotree-url`: This is the base OncoTree URL to use. If not supplied, the default will be used (http://oncotree.mskcc.org/api/)
+
+The list of OncoTree versions available are viewable [here](http://oncotree.mskcc.org/api/versions) on the dropdown menu [OncoTree home page](http://oncotree.mskcc.org/#/home).
+
+**Required Python modules:**
```

Comment:
Can we have a section called Setting up the tool which include instructions on how to download (link) and required python libraries.